### PR TITLE
Create mongo-server conection and Schema.

### PR DESCRIPTION
**Pull request by leslieroca**

### CODE REVIEW

**Comment by Reza-Lotfi** [comment]
Hey Leslie,

I think you're on the right track! There are a few things I would recommend you change before I merge this pull request. 

One really important error is that in the database index file, you created a new database called blogmodo at the top of the file. I'm assuming this is an error because you copied the code from the Technical Assessment. A better name for a database would be something like Hostels.

Another thing would be to complete your tests. We're not allowed to merge pull requests unless there is a working test suite. Currently, your test suite is not running because there are no written tests to run.

Finally, and this is really unimportant, I would recommend you add a new line at the end of each file you commit. Github doesn't like files with no empty new line at the end.

Keep up the great work!

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const Hostels = require('../database/index.js');
+const mongoose = require('../database/index.js');
```
Comment (dorktobes) [inline server/index.js]: What is the import on line 4 doing that's different than line 5? I think you meant to import `database/hostels.js` on line 4.

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const PORT = 3000;
+
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+app.get('/api/hostels', function(req, res) {
+  Hostels.find(function(err, hostels) {
```
Comment (dorktobes) [inline server/index.js]: Make sure you're handling errors :-)

File: my-test/sum.js [hunk 3]
```diff
@@ -1,3 +1,9 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+//configure({ adapter: new Adapter() });
+
+
 function sum(a, b) {
   return a + b;
 }
```
Comment (dorktobes) [inline my-test/sum.js:9]: We probably don't want to commit mock tests

File: hostelHub-mongodb /index.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+const mongoose = require('mongoose');
+const mongoUri = 'mongodb://localhost/blogmodo';
```
Comment (dorktobes) [inline hostelHub-mongodb /index.js:2]: I agree that this shouldn't be blogmodo.

**Comment by leslieroca** [comment]
Closing this pull request as I've been committing a lot of changes to the same branch with changes out of this pull request context. I'll be sending smaller pull requests from now.

**Pull request closed without merging**